tests/Feature/Api/Resources/GetLanguageLinesByGroupTest.php

```diff
@@ -78,79 +62,58 @@ public function testReturnsEmptyArrayOnEmptyGroup(): void
 
     /**
      * Asserts the route returns a valid array for a given group.
-     *
-     * @covers \App\Http\Controllers\Api\ResourcesController::getLanguageLinesByGroup
-     * @covers \App\Http\Resources\Models\LanguageLineResource::toArray
-     *
-     * @return void
      */
     public function testReturnsSuccessOnValidGroup(): void
     {
-        // Create a API language line;
-        LanguageLine::factory()->create([
-            'group' => 'api',
-            'key'   => 'EXEMPLE_KEY',
-            'text'  => [
-                'en' => 'Example Text',
-            ],
-        ]);
+        $locale = config('app.locale');
+        assert(is_string($locale));
 
-        // Assert the API line exists;
-        $this->assertDatabaseCount('language_lines', 1);
-        $this->assertDatabaseHas('language_lines', ['group' => 'api']);
```

Comment:
Personally, I believe we should trust that the factories and the test's refresh behavior did they job well. This seem like a good practice and make the tests slimmer

Reply:
It seems that I was already checking the response; but indeed I was also checking database emptiness beforehand -- I removed that!